chains/evm/calls/bridge.go

```diff
@@ -401,3 +401,65 @@ func Withdraw(client ClientDispatcher, txFabric TxFabric, gasPricer GasPricer, g
 
 	return &h, nil
 }
+
+// public function to pause: deposits, deposit executions,
+// proposal creations, and voting executions
+// https://github.com/ChainSafe/chainbridge-solidity/blob/master/contracts/Bridge.sol#L159-L165
+func Pause(client ClientDispatcher, txFabric TxFabric, gasPricer GasPricer, gasLimit uint64, bridgeAddress common.Address) (*common.Hash, error) {
+	a, err := abi.JSON(strings.NewReader(consts.BridgeABI))
+	if err != nil {
+		return nil, fmt.Errorf("could not parse bridge ABI: %w", err)
+	}
+
+	pauseInput, err := a.Pack("adminPauseTransfers")
+	if err != nil {
+		return nil, fmt.Errorf("could not prepare adminPauseTransfers input: %w", err)
+	}
+
+	h, err := Transact(
+		client,
+		txFabric,
+		gasPricer,
+		&bridgeAddress,
+		pauseInput,
+		gasLimit,
+		big.NewInt(0),
+	)
+	if err != nil {
+		return nil, fmt.Errorf("pause failed %w", err)
+	}
+	log.Debug().Str("hash", h.String()).Msgf("Pause initiated")
+
+	return &h, nil
+}
+
+// public function to unpause: deposits, deposit executions,
+// proposal creations, and voting executions
+// https://github.com/ChainSafe/chainbridge-solidity/blob/master/contracts/Bridge.sol#L167-L173
+func Unpause(client ClientDispatcher, txFabric TxFabric, gasPricer GasPricer, gasLimit uint64, bridgeAddress common.Address) (*common.Hash, error) {
+	a, err := abi.JSON(strings.NewReader(consts.BridgeABI))
+	if err != nil {
+		return nil, fmt.Errorf("could not parse bridge ABI: %w", err)
+	}
+
+	unpauseInput, err := a.Pack("adminUnpauseTransfers")
+	if err != nil {
+		return nil, fmt.Errorf("could not prepare adminUnpauseTransfers input: %w", err)
+	}
+
+	h, err := Transact(
+		client,
+		txFabric,
+		gasPricer,
+		&bridgeAddress,
+		unpauseInput,
+		gasLimit,
+		big.NewInt(0),
+	)
+	if err != nil {
+		return nil, fmt.Errorf("unpause failed %w", err)
+	}
+	log.Debug().Str("hash", h.String()).Msgf("Unpause initiated")
```

Comment:
Doesn't really make sense to log here and in `Pause` if we are gonna log hash in command as well and 
also `Transact` logs tx hash anyway.